.travis.yml

```diff
@@ -16,16 +16,19 @@ install:
   - pip install https://github.com/biocore/mustached-octo-ironman/archive/master.zip
   - pip install https://github.com/biocore/qiime/archive/master.zip --no-deps
   - pip install .
+  # upgrade click, older versions of click do not have __version__
+  - pip install -U click
 script:
   - export MOI_CONFIG_FP=`pwd`/qiita_core/support_files/config_test.txt
   - ipython profile create qiita_general --parallel
   - qiita_env start_cluster qiita_general
   - qiita_env make --no-load-ontologies
   - nosetests --with-doctest --with-coverage
-  - flake8 qiita_* setup.py scripts/qiita scripts/qiita_env
+  - flake8 qiita_* setup.py scripts/qiita scripts/qiita_env scripts/qiita_test_install
   # we need to run the test suite from setup.py for coveralls to grab the info
   # - coverage run setup.py test
   # - coverage report -m
+  - qiita_test_install
```

Comment:
Should this be moved up to second thing called in the script? That way we have all the info before tests are run, so we can fix things as they error if needed.